### ISSUE TITLE
NDPS Missed File Netowrk Issue

### ISSUE THREAD

**Issue by mbharathm** [beezlabs-org/l3-support#25]

**Comment by mbharathm** [comment]
Do we still have this issue active and occurring recently @AmitRanjanBasu ?
I believe it is one month old and we have informed them that it is networked based issue. We couldn't reproduce this as well. 
But I heard from @kruthik-nivas that we can handle this error.
Please confirm if there is a solution, else confirm the closure of this issue?

**Comment by kruthik-nivas** [comment]
Yeah, we can handle this issue but we have planned this along with the UAT of the next cluster. So I prefer we can keep this open until this is moved to productions
